TeamCharles/initial-site
# 106-UpdatePriceTypeToDecimal :: Updated model for product price to be a decimal

## Status

**READY**
## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.
## Description

Changes the model for product price from float to decimal.
## Resolves Issue Number
#106
## Deploy Notes

Delete Migrations
Delete Database

``` sh
git pull --prune
rm -rf Migrations/
rm banganzon.db
donet ef migrations add Initial
dotnet ef database update
git checkout 106-UpdatePriceTypeToDecimal
dotnet run
```
## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
1.  Delete database
2. Update and run migrations
3. dotnet run
4. Should see pricing with two decimals
## Impacted Files in Application

List general components of the application that this PR will affect:
- Data/DbInitializer.cs
- ViewModels/Cart.cs
- Models/Product.cs
- Migrations/
